core/src/tangram.h

```diff
@@ -139,6 +139,10 @@ void loadScene(const char* _scenePath, bool _setPositionFromScene = false);
 
 void runOnMainLoop(std::function<void()> _task);
 
+void setSceneComponent(const char* componentName, const char* value);
```

Comment:
Can we make it more explicit that updates are deferred? Something like `queueSceneUpdate` sounds more honest about what's going on here.